kore/src/Kore/Exec/GraphTraversal.hs

```diff
@@ -48,8 +48,8 @@ data TransitionResult a
     | -- | final state (e.g., goal state reached, side conditions hold)
       Final a
     | -- | not stuck, but also not final (maximum depth reached before
-      -- finishing the proof). Provides the "next" states for the result.
-      Stopped [a]
+      -- finishing the proof). Provides current and "next" states for the result.
+      Stop a [a]
```

Comment:
Just curious, doesnt matter either way, but why the change?

Reply:
(i.e. Stopped -> Stop)

Reply:
I thought `Stop/Stopped` in analogy to `Stuck _ :: TransitionResult` => `GotStuck _ _ :: TraversalResult`. 
`Stopped` is a good name for the `TraversalResult`, and we cannot have two `Stopped` constructors :man_shrugging:

Reply:
Fair enough

Reply:
LGTM btw